frontend/src/components/pages/Scheduling/SelectDateTime.tsx

```diff
@@ -420,54 +447,28 @@ const SelectDateTime = ({
           mb="3em"
         >
           <FormLabel fontWeight="600">Proposed end date</FormLabel>
-          <Input
-            isDisabled={isBeingEdited}
-            value={recurringEndDate}
-            onChange={(e) => handleChangeRecurringDate(e.target.value)}
-            placeholder="MM/DD/YYYY"
-            maxWidth="350px"
-            size="lg"
-          />
+          <FormLabel fontWeight="400">Date</FormLabel>
+          <SimpleGrid columns={2} columnGap={16} rowGap={6} w="full">
```

Comment:
is there any way to block out dates earlier than today? just out of curiousity for some potential errors to leak in

and we can't choose the next month - not sure if this is the wanted behaviour?

Reply:
yeah ideally the calendar is two weeks _starting_ today